cli/rootflags/flags.go

```diff
@@ -0,0 +1,30 @@
+package rootflags
+
+import (
+	"errors"
+	"flag"
+	"os"
+)
+
+type Flags struct {
```

Comment:
NIt: I don't think this deserves its own package under `cli`. I'd call it `RootFlags` and put it directly in `cli` instead.

Reply:
agreed.